mappings/net/minecraft/util/CommonColors.mapping

```diff
@@ -0,0 +1,8 @@
+CLASS net/minecraft/unmapped/C_oauvuath net/minecraft/util/CommonColors
```

Comment:
`Common` is a bit of a mojmap thing, I think we could get away with just `Colors` here

Reply:
i actually was thinking of like, considering adopting that name; screentext after all isn't really a better name than commontext (commoncomponents)